panel/src/components/Forms/Input/DateInput.vue

```diff
@@ -149,7 +149,7 @@ export default {
      * Separator from `display` format
      */
     separator() {
-      return this.display.match(/[\W]/)[0];
+      return (this.display.match(/[\W]/) ?? [])[0] ?? "";
```

Comment:
`this.display.match(/[\W]/)?.[0] ?? ""` should work as well. 👀

Reply:
Yes, this is more cleaner 👍

Reply:
@johannschopplich that looks a lot better indeed. I'm still not super familiar with the optional operator possibilities.

Reply:
@bastianallgeier That's why I wanted to gently point this out to you when I happened to look in the commit. Glad it to hear it looks good to you.